book/actix-plugin.md

```diff
@@ -193,6 +193,53 @@ async fn my_handler() -> Result<(), MyError> {
 }
 ```
 
+#### Defining security
+
+Use `Apiv2Security` derive macro for struct used as handler parameter to have this handler marked as requiring authorization.
+
+```rust
+use paperclip::actix::Apiv2Security;
+
+#[derive(Apiv2Security)]
+#[openapi(
+  apiKey,
+  in = "header",
+  name = "Authorization",
+  description = "Use format 'Bearer TOKEN'"
+)]
+pub struct AccessToken;
+
+impl FromRequest for Accesstoken { /*...*/ }
+
+#[api_v2_operation]
+async fn my_handler(access_token: AccessToken) -> Result {
+    /*...*/
+}
+```
+
+First parameter is the type of security, currently supported types are "apiKey" and "oauth2". Possible parameters are `alias`, `description`, `name`, `in`, `flow`, `auth_url`, `token_url` or `parent`.
+
+Use `alias` parameter if you need to have two different security definitions of the same type.
+
+If you need to define scopes for `oauth2`, use `parent` attribute:
+
+```rust
+#[derive(Apiv2Security, Deserialize)]
+#[openapi(
+  oauth2,
+  auth_url = "http://example.com/",
+  token_url = "http://example.com/token",
+  flow = "password"
+)]
+struct OAuth2Access;
+
+#[derive(Apiv2Security, Deserialize)]
+#[openapi(parent = "oauth2", scopes("pets.read", "pets.write"))]
```

Comment:
What is the motivation behind `parent = "oauth2"`? Can't we have `parent = OAuth2Access`? Because, it shouldn't be the case that `OAuth2Access` must be used in some handler. If we had the latter, then we can generate the definition if it doesn't already exist.